pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/embeddings/AbstractHuggingFaceEmbeddingService.java

```diff
@@ -43,6 +43,8 @@ public abstract class AbstractHuggingFaceEmbeddingService<IN, OUT>
    */
   public static final String URL_PREFIXES_SYSTEM_PROP = "ALLOWED_HF_URLS";
 
+  private static final String DLJ_BASE_URL = "djl://ai.djl.huggingface.pytorch/sentence-transformers/";
```

Comment:
"djl://" should be enough. or "djl://ai.djl.huggingface.pytorch/" if you want to reuse it to build modelUrl.

"sentence-transformers" is not a tag/filter but a part of the model name in this case. E.g. something like "flax-sentence-embeddings/stackoverflow_mpnet-base" can be valid.

Reply:
updated